deployment/fedora-package-x64/pkg-src/jellyfin.spec

```
@@ -29,7 +29,7 @@ Requires:       libcurl, fontconfig, freetype, openssl, glibc libicu
 # COPR @dotnet-sig/dotnet
 BuildRequires:  dotnet-sdk-2.2
 # RPMfusion free
-Requires:       ffmpeg
+Requires:       ffmpeg <4.1 or jellyfin-ffmpeg
```

`rpmlint` doesn't like this.
```suggestion
Requires:       (ffmpeg < 4.1 or jellyfin-ffmpeg)
```

---

This will not work on CentOS since it has ancient packages `rpm-4.11.3`  while fedora has `rpm-4.14.2` so the "New" RPM Boolean dependencies will probably result in the package failing to install on CentOS.

RPM Fusion also provides the right ffmpeg versions:

- CentOS 7 ffmpeg-3.4.5-2
- Fedora 28 ffmpeg-4.0-1
- Fedora 29 ffmpeg-4.0.2

---

Thanks @Wuerfelbecher, I'll leave this as a Debian-only change for now.